### PR TITLE
Cava wallbash Feature 

### PR DESCRIPTION
**Pull request by kRHYME7**
Override cava gradient config using wallbash

https://github.com/prasanthrangan/hyprdots/assets/53417443/bbc453db-fe2d-4d5a-84be-f02b2bdf3033

image

image
image

### CODE REVIEW

File: Configs/.config/hypr/scripts/wallbashcava.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+ScrDir=$(dirname "$(realpath "$0")")
+. $ScrDir/globalcontrol.sh
+CAVA_CONF="/home/khing/.config/cava/config"
+WALL_DCOL="/home/khing/.config/cava/Wall-Dcol"
```
Comment (zer0cha0s) [inline Configs/.config/hypr/scripts/wallbashcava.sh:6]: Need to change your home dir to $HOME

**Review by zer0cha0s — CHANGES_REQUESTED** [review]
Nice work as always dude, minor change to home dir required to get the script to work for me and also had to give execute permissions to the script but i did copy the files over manually as to not break my set up. (Dont know if there is anything in the install script that applies permissions when it sets up and copies the files over.)

**Comment by kRHYME7** [comment]
Thanks! I totally forgot to revert this

File: Configs/.config/hypr/scripts/wallbashcava.sh (hunk 1 above)
Comment (kRHYME7) [reply Configs/.config/hypr/scripts/wallbashcava.sh]: Thanks! I totally forgot to revert while testing

**Review by zer0cha0s — APPROVED** [review]
Can confirm changes work as advertised, no issues found.

File: Configs/.config/hypr/wallbash/color.dcol
Comment (prasanthrangan) [inline Configs/.config/hypr/wallbash/color.dcol:1]: This file seems unnecessary, so i'll remove this.

**Review by prasanthrangan — APPROVED** [review]
looks great, thank you so much!

**Pull request merged**